### ISSUE TITLE
Angular template - modal backdrop is being cut if modal height is higher than 100%

### ISSUE THREAD

**Issue by thiag0coelho** [abpframework/abp#2487]
![image](https://user-images.githubusercontent.com/6138475/71544889-2061c180-2963-11ea-8bf3-eeccf4b8885c.png)
When form is too large modal backdrop is being cut, it's max height is 100%

Tried to make a PR to fix this issue changing scss like so

```
.modal-backdrop {
    position: fixed;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    z-index: 1040;
    background-color: #000;
    opacity: 0.6;
}
```

But then scrolling on backdrop does not work. I'll leave it to you

**Comment by mehmet-erim** [comment]
Hi @thiag0coelho
I haven't seen any problem.
Which version of `@abp/ng.theme.shared` is installed in your application?

The current version is 1.1.1 and `.modal-backdrop` styles like this:
```css
.modal-backdrop {
  position: fixed;
  top: 0;
  left: 0;
  width: calc(100% - 7px);
  height: 100%;
  background-color: rgba(0, 0, 0, 0.6);
  z-index: 1040;
}
```

**Comment by thiag0coelho** [comment]
Currently using 1.1.1. Uploaded a gif to help reproduce the issue. You have to decrease the screen size enough until the scroll appears..
Hope that helps

![Hpe065Orqj](https://user-images.githubusercontent.com/6138475/71648093-94db8e00-2cde-11ea-84ec-fd49c10a385f.gif)

![image](https://user-images.githubusercontent.com/6138475/71648031-c011ad80-2cdd-11ea-94d1-93822d65bfbf.png)

**Comment by thediaval** [comment]
We tried reproducing the problem, it is working fine in the latest version. You can remove `node_modules` folder and `yarn.lock` file and re-install packages to get the latest version.

Also did some changes, it will be available in next version: 4b5c6a28b75d2ceb88888ee49f33c5b22ac47700

**Comment by thiag0coelho** [comment]
That worked. Thanks!
